include/mosquitto.h

```diff
@@ -1665,6 +1665,18 @@ libmosq_EXPORT void mosquitto_user_data_set(struct mosquitto *mosq, void *obj);
  */
 libmosq_EXPORT void *mosquitto_userdata(struct mosquitto *mosq);
 
+/*
+ * Function: mosquitto_delayed_puback_set
```

Comment:
```suggestion
 * Function: mosquitto_delayed_puback
```

why the set postfix?

Reply:
```suggestion
 * Function: mosquitto_delay_puback
```